test/test_quantization.py

```diff
@@ -519,8 +519,34 @@ def test_quantized_rnn(self):
 
         torch.testing.assert_allclose(output_int8, ref_out)
         self.assertEqual(output_int8, ref_out)
-        for out, ref in zip(final_hiddens_int8, ref_hid):
-            torch.testing.assert_allclose(out, ref)
+        for out_val, ref_val in zip(final_hiddens_int8, ref_hid):
+            torch.testing.assert_allclose(out_val, ref_val)
+
+        class ScriptWrapper(torch.nn.Module):
```

Comment:
Can we reuse the `model` above as `model = LSTMDynamicModel().eval()`, instead of creating a new PyTorch module wrapper? It is defined in https://github.com/pytorch/pytorch/blob/master/test/common_quantization.py#L182-L190. I am OK with the current code if you think this is more consistent in the unit test.

Reply:
No, this is to ensure that the serialization and scripting functionality works even in a nested context. This is a case that has often been overlooked in the past